core/src/main/java/hudson/security/GlobalSecurityConfiguration.java

```diff
@@ -59,26 +59,42 @@
 public class GlobalSecurityConfiguration extends ManagementLink implements Describable<GlobalSecurityConfiguration> {
     
     private static final Logger LOGGER = Logger.getLogger(GlobalSecurityConfiguration.class.getName());
+    
+    protected Jenkins getJenkins() {
+        return Jenkins.getInstance();
+    }
 
     public MarkupFormatter getMarkupFormatter() {
-        return Jenkins.getInstance().getMarkupFormatter();
+        return getJenkins().getMarkupFormatter();
```

Comment:
loads of findbugs for potential NPE with this style as `getInstance()` may return null